pkg/kubelet/cm/event_dispatcher.go

```diff
@@ -108,22 +110,24 @@ func (ed *eventDispatcher) dispatchEvent(pod *api.Pod, cgroupPath string, kind l
 
 		glog.Infof("Dispatching to event handler: %s", name)
 		reply, err := client.Notify(ctx, ev)
+		replies = append(replies, reply)
 		if err != nil {
-			return err
+			return replies, err
 		}
 		if reply.Error != "" {
-			return errors.New(reply.Error)
+			return replies, errors.New(reply.Error)
 		}
-	}
 
-	return nil
+	}
+	// TODO: support more than one isolator return list of replies
```

Comment:
Didn't understand this comment (looks like you are already returning multiple replies)